lib/hexpm_web/templates/docs/self_hosting.html.md

```diff
@@ -0,0 +1,360 @@
+## Self-hosting
+
+In this guide we'll learn how to serve Hex packages using your own server. First, though, some background on what a Hex server really is.
+
+### Hex specifications
+
+A Hex server must follow [Hex specifications](https://github.com/hexpm/specifications). An example server, and a reference implementation, is <https://github.com/hexpm/hexpm> that powers <https://hex.pm>. The Hex team also maintains a lower-level library, <https://github.com/hexpm/hex_core>, that you can use to build and interact with Hex servers.
```

Comment:
```suggestion
A Hex deployment must follow the [Hex specifications](https://github.com/hexpm/specifications). <https://hex.pm> is powered by <https://github.com/hexpm/hexpm> and related services such as <https://github.com/hexpm/hexdocs>. While you can use these projects to run your own Hex infrastructure it is generally not recommended because they include many features and complexity not required by the average deployment. The Hex team also maintains a lower-level library, <https://github.com/hexpm/hex_core>, that you can use to build and interact with Hex services.
```